roottest/cling/offset/runvbase.C

```diff
@@ -35,7 +35,7 @@ struct Bottom : public Mid1, Mid2
 #pragma warning(push)
 #pragma warning(disable : 4584)
 #endif
-#ifdef __CLING__
+#if defined(__CLING__) || defined(__clang__)
```

Comment:
I think
```suggestion
#if defined(__clang__)
```
should be sufficient